tls/s2n_resume.h

```diff
@@ -20,8 +20,7 @@
 #include "stuffer/s2n_stuffer.h"
 
 #define S2N_STATE_LIFETIME_IN_NANOS     54000000000000      /* 15 hours */
-#define S2N_TLS12_STATE_SIZE_IN_BYTES   (1 + 8 + 1 + S2N_TLS_CIPHER_SUITE_LEN + S2N_TLS_SECRET_LEN)
-
+#define S2N_TLS12_STATE_SIZE_IN_BYTES   (1 + 8 + 1 + S2N_TLS_CIPHER_SUITE_LEN + S2N_TLS_SECRET_LEN + 1)
```

Comment:
You removed the check in the de-serialize method that used this-- is the macro still necessary?

Reply:
This macro is pretty widely used in the code and in our tests; I'm not sure it's worth the effort to remove it.